example/books/books.proto

```diff
@@ -3,12 +3,15 @@ syntax = "proto3";
 package books;
 
 import "github.com/kitt-technology/protoc-gen-graphql/graphql/graphql.proto";
-import "github.com/kitt-technology/protos-common/common.proto";
+import "github.com/kitt-technology/protoc-gen-graphql/example/common/common.proto";
 import "google/protobuf/wrappers.proto";
 import "google/protobuf/timestamp.proto";
 
 option go_package = "example/books;books";
 
+option (graphql.common_go_import) = "github.com/kitt-technology/protoc-gen-graphql/example/common";
```

Comment:
Might be able to get rid of this? Its the same as the `go_package` in the common file so might be able to figure out?